crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,56 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+  border: 1px;
+  border-color: $brand-gray;
+  border-left: 7px;
+  border-radius: 5px;
+  border-style: solid;
+  box-shadow: 1px 1px 0 $gray-lighter;
+  margin: 5px;
+}
+
+.panel-heading+.list-group .list-group-item:first-child {
+  border-top-width: 1px;
+}
+
+.panel>.list-group:last-child .list-group-item:last-child, .panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{
+  border-bottom: 1px solid #ccc; ;
+}
+
+.status-border-unready {
+  animation-duration: 1400ms;
+  animation-iteration-count: infinite;
+  animation-name: blink;
+  border-left-color: $brand-gray !important;
+}
+
+
+.status-border-ready {
+  border-left-color: #599517 !important;
+}
+
+.status-border-failed {
+  border-left-color: #ff001f !important;
+}
+
+.status-border-unknown {
+  border-left-color: $brand-gray !important;
+}
+
+.status-border-upgrade {
+  border-left-color: #4a90e2 !important;
```

Comment:
Color literals like `#4a90e2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used